### DISCUSSION TITLE
Gallery plugin displacement of images

### DISCUSSION THREAD

**Discussion by ghost** [datenstrom/community#200] [Report a bug] [answered]
Hi!
and thanks for Yellow! It's awesome I really enjoy using it!
Just few hours of coding and my new site is up and running, good job!

The Gallery plugin displaces the first row of images with 1 or 2 pixels why ?
Can't find out what is triggering this?

Any hints?

**Comment by markseuffert** [comment]
Hi Niels, 

I think there's a non-existing image at the beginning. What's the [gallery shortcut](https://github.com/annaesvensson/yellow-gallery) you are using?

**Comment by ghost** [comment]
Hi Mark,
the code is: [gallery gallery - 150]

The images is in a subfolder like:
media/images/gallery

I've checked that there's no unintended white space in the markdown file pages.txt.

**Comment by markseuffert** [comment]
Looks like you found a bug, the shortcut generates the wrong HTML code. Sorry about that. I'll have a look at the code and see what I can find out. In the meantime, you can use a shortcut like this:

`[gallery gallery/.*jpg - 150]`. 

Hope this works better.

**Comment by ghost** [comment]
Thanks a lot!
That did the trick!

**Comment by markseuffert** [comment, marked as answer]
Thanks a lot for reporting it. Oops, this shouldn’t have happened. With the latest core plugin it should work no matter which shortcut you use. I also added some more examples to the documentation, hopefully this makes it easier to get a website up and running.

Bug reports and questions are always welcome.
